desktop/languages/de.json

```diff
@@ -0,0 +1,77 @@
+{
+	"update": "Auf Aktualisierung prüfen",
+	"settings": {
+		"hosting": {
+			"heading": "Hosting",
+			"subheading": "Verbinde dich mit einem verfügbaren Webhost, um deine Seiten im Internet zu veröffentlichen",
+			"connect": "Mit Host verbinden",
+			"host": {
+				"back": "Zurück zu den Hosts",
+				"heading": "Erstelle einen <a href='https://vercel.com/account/tokens'>Token</a> und gib ihn ein, um die Verbindung zum Webhost abzuschließen",
```

Comment:
It seems that the part about Netlify is missing, because there is only a Vercel url.